src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1818,15 +1820,29 @@ internal enum ErrorCode
         ERR_AddressOfMethodGroupInExpressionTree = 8810,
         ERR_CannotConvertAddressOfToDelegate = 8811,
         ERR_AddressOfToNonFunctionPointer = 8812,
-        ERR_FeatureNotAvailableInVersion9 = 8813,
 
-        // Codes 8813, 8814, 8815, 8816 used by features/module-initializers
+        [Obsolete("used by features/module-initializers")]
+        ERR_8813 = 8813,
+
+        [Obsolete("used by features/module-initializers")]
+        ERR_8814 = 8814,
+
+        [Obsolete("used by features/module-initializers")]
+        ERR_8815 = 8815,
+
+        [Obsolete("used by features/module-initializers")]
+        ERR_8816 = 8816,
```

Comment:
Do we need `[Obsolete]` attributes? Would the comment plus placeholders be sufficient?
```
// Codes 8813, ... used by features/module-initializers
ERR_8813 = 8813,
...
```